sdk/src/main/java/io/dapr/actors/client/IActorProxyFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.actors.client;
+
+import io.dapr.actors.ActorId;
+
+public interface IActorProxyFactory {
```

Comment:
Please, remove this interface. We will use the Builder pattern, similar to what we have in the http client.

Reply:
https://azure.github.io/azure-sdk/java_design.html#service-client-builder